src/core/MOM_barotropic.F90

```diff
@@ -1272,12 +1272,12 @@ subroutine btstep(U_in, V_in, eta_in, dt, bc_accel_u, bc_accel_v, forces, pbce,
         CS%IDatv(i,J) = 0.0
       elseif (integral_BT_cont) then
         CS%IDatv(i,J) = CS%dx_Cv(i,J) / (max(find_dvhbt_dv(vbt(i,J)*dt, BTCL_v(i,J)), &
-                                             CS%dx_Cv(i,J)*Htot_avg) )
+                                             CS%dx_Cv(i,J)*Htot_avg) ) * GV%Z_to_H
```

Comment:
Please also modify the v-point calculations on lines 1275 and 1278, analogously to what was done for the u-points on lines 1248 and 1251.